### PR TITLE
fix(create-analog): fixes postcss.config.cjs issue

### PR DESCRIPTION
**Pull request by pavankjadda**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/analogjs/analog/blob/main/CONTRIBUTING.md#-commit-message-guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Which package are you modifying?

- [ ] vite-plugin-angular
- [ ] vite-plugin-nitro
- [ ] astro-angular
- [x] create-analog
- [ ] router
- [ ] platform
- [ ] content
- [ ] nx-plugin
- [ ] trpc

## What is the current behavior?

Closes #

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: packages/create-analog/index.js [hunk 1]
```diff
@@ -333,7 +333,7 @@ function addTailwindDirectives(write, filesDir) {
 
 function addPostCssConfig(write, filesDir) {
   write(
-    'postcss.config.js',
```
Comment (yousefalaqra) [inline packages/create-analog/index.js:336]: Do you need to change this part as well?

function addTailwindConfig(write, filesDir) {
  write(
    'tailwind.config.js',
    fs.readFileSync(path.join(filesDir, `tailwind.config.js`), 'utf-8')
  );
}

File: packages/create-analog/index.js [hunk 2]
```diff
@@ -333,14 +333,14 @@ function addTailwindDirectives(write, filesDir) {
...
+    'postcss.config.cjs',
     fs.readFileSync(path.join(filesDir, `postcss.config.js`), 'utf-8')
```
Comment (brandonroberts) [inline packages/create-analog/index.js:337]:
```suggestion
    fs.readFileSync(path.join(filesDir, `postcss.config.cjs`), 'utf-8')
```

File: packages/create-analog/index.js [hunk 3]
```diff
@@ -333,14 +333,14 @@ function addTailwindDirectives(write, filesDir) {
...
   );
 }
 
 function addTailwindConfig(write, filesDir) {
   write(
-    'tailwind.config.js',
+    'tailwind.config.cjs',
     fs.readFileSync(path.join(filesDir, `tailwind.config.js`), 'utf-8')
```
Comment (brandonroberts) [inline packages/create-analog/index.js:344]:
```suggestion
    fs.readFileSync(path.join(filesDir, `tailwind.config.cjs`), 'utf-8')
```

**Review by brandonroberts — CHANGES_REQUESTED** [review]

**Review by brandonroberts — APPROVED** [review]

**Comment by brandonroberts** [comment]
Thanks! @allcontributors add @pavankjadda for code

**Pull request merged**